frontend/app/components/company-uebersicht-edit.js

```diff
@@ -1,64 +0,0 @@
-import { inject as service } from '@ember/service';
```

Comment:
Ich habe diese Komponente "entdeutscht" und in `company-overview-edit` umbenannt.